package.json

```diff
@@ -2,6 +2,11 @@
   "name": "@openedx/brand-openedx",
   "version": "1.0.0-semantically-released",
   "description": "The default branding and SASS theme package containing for Open edX applications. This package is designed to be copied and customized.",
+  "scripts": {
+    "build-tokens": "paragon build-tokens --source ./paragon/tokens/ --build-dir ./paragon/build -t dark -t light",
```

Comment:
There is no dark theme variant yet; should we avoid including `-t dark` until there is an official dark theme variant?

Reply:
I use this as a template to demonstrate how to use it with multiple themes. Despite paragon does not have more variants besides light, that does not limit brand package to create those. 
However following your suggestion about having a unique command could help to manage this